### PR TITLE
Add posts - functionality

### PR DESCRIPTION
**Pull request by smncrm**

### CODE REVIEW

File: src/main/java/mops/foren/infrastructure/web/ForenController.java [hunk 1]
```diff
@@ -26,6 +23,7 @@
     private ThreadService threadService;
     private PostService postService;
     private ForumForm form = new ForumForm("", "");
+    private PostForm postForm = new PostForm("");
```
Comment (niruc100) [inline src/main/java/mops/foren/infrastructure/web/ForenController.java:26]: why are they empty?

File: src/main/java/mops/foren/infrastructure/web/ForenController.java [hunk 2]
```diff
@@ -137,24 +135,40 @@ public String enterATopic(@PathVariable String forenID,
...
+     */
+    @PostMapping("/post/newPost")
+    @RolesAllowed({"ROLE_studentin", "ROLE_orga"})
+    public String newPost(KeycloakAuthenticationToken token, @ModelAttribute PostForm postForm,
+                          @RequestParam("threadId") Long threadIdL) {
+        ThreadId threadId = new ThreadId(threadIdL);
+        User user = this.userService.getUserFromDB(token);
+        Post post = postForm.getPost(user, threadId);
+        this.threadService.addPostInThread(threadId, post);
+        return "redirect:/thread?threadId=" + threadIdL;
```
Comment (niruc100) [inline src/main/java/mops/foren/infrastructure/web/ForenController.java]: rename "threadIdL" to e.g. threadIdLong or numericalThreadID

File: src/main/java/mops/foren/infrastructure/web/PostForm.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import mops.foren.domain.model.User;
+
+@Value
+public class PostForm {
+    private final String postContent;
+
+
+    /**
+     * This method maps return the corresponding Post.
+     *
```
Comment (niruc100) [inline src/main/java/mops/foren/infrastructure/web/PostForm.java]: refactor description

**Review by niruc100 — APPROVED** [review]
small changes

File: src/main/java/mops/foren/infrastructure/web/PostForm.java (hunk 3 above)
Comment (smncrm) [reply src/main/java/mops/foren/infrastructure/web/PostForm.java]: done

File: src/main/java/mops/foren/infrastructure/web/ForenController.java (hunk 2 above)
Comment (smncrm) [reply src/main/java/mops/foren/infrastructure/web/ForenController.java]: done

File: src/main/java/mops/foren/infrastructure/web/ForenController.java (hunk 1 above)
Comment (smncrm) [reply src/main/java/mops/foren/infrastructure/web/ForenController.java:26]: good point, to be decided in the future. In general, for all forms that we use.

**Pull request merged**